packages/editor-ui/package.json

```diff
@@ -65,7 +65,6 @@
     "n8n-workflow": "workspace:*",
     "normalize-wheel": "^1.0.1",
     "pinia": "^2.1.6",
-    "prettier": "^2.8.8",
```

Comment:
wouldn't this cause the `import/no-extraneous-dependencies` linting errors?

Reply:
True! Restored and upgraded.